src/main/resources/static/css/style.css

```diff
@@ -1,5 +1,8 @@
 html {
     font-size: 18px;
+    -webkit-user-select: none;
```

Comment:
Tohle je tam kvůli čemu?

Reply:
znemožnění výběru textu - je to bod 6.1 z toho tvojeho odkazu na issues

Reply:
a, chyba - mělo to být v body, nejak jsem se upsala - opravim to

Reply:
To uz asi nehraje roli, si myslim.

Reply:
Nedával bych.

Reply:
Souhlasim.